src/grid/interpolate.jl

```diff
@@ -197,17 +208,27 @@ end
 
 
 """
-    function interpGrid(data, xgrid, grid)
-
-linear interpolation of data(grid[1:end]), return a Vector
+    function interpGrid(data, xgrid, grid; axis=1)
+For 1D data, do interpolation of data(grid[1:end]), return a Vector.
+For ND data, do interpolation of data(grid[1:end]) at given axis, return data of same dimension.
 
 #Arguments:
 - xgrid: one-dimensional grid of x
 - data: one-dimensional array of data
 - grid: points to be interpolated on
+- axis: axis to be interpolated in data
+- interpstyle: by default use optimized method; use linear interp if :linear
 """
-function interpGrid(data, xgrid::T, grid) where {T}
-    interpGrid(InterpStyle(T), data, xgrid, grid)
+function interpGrid(data, xgrid::T, grid; axis=1, interpstyle=:default) where {T}
```

Comment:
interpGrid ->interp1DGrid

Reply:
it's renamed to interp1DGrid. please ignore the typo in the message of new commit